### ISSUE TITLE
Code lens run/debug test only works if the test is in the root folder of the test project

### ISSUE THREAD

**Issue by eamodio** [OmniSharp/omnisharp-vscode#453]
## Environment data

`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-preview1-002702)

Product Information:
 Version:     1.0.0-preview1-002702
 Commit Sha:  6cde21225e

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
```

VS Code version: 
`Version 1.3.0-insider (1.3.0-insider) 93e65224d7598017bfa0b188a0d82b66a2ee9698`

C# Extension version:
`1.1.6`
## Steps to reproduce

Have a test project, with a test that isn't in the root of that project
Use the code lens run/debug test
## Expected  behavior

Test either runs or debugs based on the choice
## Actual behavior

The following exception is throw:

```
dotnet-test Error: 0 : System.InvalidOperationException: /Users/foo/code/foo.Tests/TestCaseFolder/project.json does not exist.
   at Microsoft.DotNet.Tools.Test.TestCommand.GetProjectPath(String projectPath)
   at Microsoft.DotNet.Tools.Test.TestCommand.DoRun(String[] args)
```

**Comment by gregg-miskelly** [comment]
@troydai FYI

**Comment by troydai** [comment]
ah, yup. OmniSharp should move up the folder to find actual project root. It's a bug in `OmniSharp-roslyn` I will file an issue there to trace this problem.

**Comment by troydai** [comment]
I'll assign this to me and keep it open just in case if there is fix required on the C# extension side.

**Comment by troydai** [comment]
@eamodio thanks for filing this issue.

**Comment by DustinCampbell** [comment]
@troydai -- let's get the last OmniSharp changes in and then we'll produce a new release that we can take into the extension.

**Comment by troydai** [comment]
@DustinCampbell yup

**Comment by troydai** [comment]
Assign to @DustinCampbell

**Comment by DustinCampbell** [comment]
This was fixed in 1.9.
